modules/kubernetes-addons/aws-node-termination-handler/main.tf

```diff
@@ -29,6 +29,7 @@ resource "aws_autoscaling_group_tag" "aws_node_termination_handler_tag" {
   }
 }
 
+#tfsec:ignore:aws-sqs-enable-queue-encryption
```

Comment:
SSE encryption is enabled, but looks like tfsec wants CMK encryption so silencing error for now